GDJS/Runtime/layer.ts

```diff
@@ -30,6 +30,12 @@ namespace gdjs {
 
       this._cameraX = this.getWidth() / 2;
       this._cameraY = this.getHeight() / 2;
+      if (this.getCameraType() === gdjs.RuntimeLayerCameraType.ORTHOGRAPHIC) {
+        this._cameraZ =
+          (this._initialCamera3DFarPlaneDistance +
+            this._initialCamera3DNearPlaneDistance) /
+          2;
+      }
```

Comment:
The camera Z position doesn't change according to the zoom and stayed at 0 which makes most of the objects behind the far plane.

Reply:
Should this comment be in the code?